stock_cycle_count/models/stock_inventory.py

```diff
@@ -17,7 +17,8 @@ class StockInventory(models.Model):
     def _compute_inventory_accuracy(self):
         zero_inventories = self.filtered(
             lambda i: not i.line_ids and i.state == 'done')
-        zero_inventories.inventory_accuracy = PERCENT
+        for inv in zero_inventories:
+            inv.inventory_accuracy = PERCENT
```

Comment:
And the only way to fix it is this commit (https://github.com/OCA/stock-logistics-warehouse/pull/346/commits/2b4103365fa4c31e041fda3f8ee387c1aa967e97) which I think leave us in a position very similar to the starting one and I hardly see any performance improvement. What do you think?

Reply:
No, in fact. I didn't realized.

Indeed you cannot write on compute. Wrong! You have to change recordset fields one by one(the save is done at the end).

The improvement is only valid outside computes. Sorry.

Reply:
I didn't get a notification of this answer. No worries, it is always good to learn :wink:

Reply:
@rousseldenis reverted to initial state, could you please review again?